tests/service-catalog/test/binding_usage_test.go

```diff
@@ -409,7 +454,13 @@ func (ts *TestSuite) createAndWaitBindingUsage(bindingName, sbuName, envPrefix s
 		}
 	}
 
-	_, err := usageCli.Create(sbu)
+	err := wait.Poll(time.Second, time.Minute, func() (bool, error) {
```

Comment:
you can use const here:
```suggestion
	err := wait.Poll(time.Second, timeoutPerStep, func() (bool, error) {
```